crates/bindings/c/src/lib.rs

```diff
@@ -3,11 +3,26 @@
 
 mod types;
 
+use std::{sync::{Once, Mutex}, borrow::BorrowMut};
+
+use tokio::runtime::Runtime;
 use types::{
     blob, libsql_connection, libsql_connection_t, libsql_database, libsql_database_t, libsql_row,
     libsql_row_t, libsql_rows, libsql_rows_future, libsql_rows_future_t, libsql_rows_t,
 };
 
+static mut RT: Option<Mutex<Runtime>> = None;
+static INIT: Once = Once::new();
+
+fn get_rt<'a>() -> &'a Mutex<Runtime> {
```

Comment:
You might find lazy_static useful for exactly that purpose: https://docs.rs/lazy_static/latest/lazy_static/

Reply:
That worked. Thanks